templates/default/exabgp.conf.erb

```diff
@@ -15,7 +15,8 @@ neighbor <%= @neighbor_ipv4 %> {
   }
 
   static {
-    route <%= @route_ipv4 %> {
+    <% @route_ipv4.each do |ip| %>
```

Comment:
I suggest somehow to check if it is array/interarable for backwards compatability